pennylane/__init__.py

```diff
@@ -40,9 +40,14 @@
 from pennylane.optimize import *
 from pennylane.qnode import QNode, qnode
 from pennylane.templates import broadcast, layer, template
-from pennylane.transforms import draw, measurement_grouping, metric_tensor
-from pennylane.transforms.adjoint import adjoint
-from pennylane.transforms.control import ctrl, ControlledOperation
+from pennylane.transforms import (
+    adjoint,
+    draw,
+    ControlledOperation,
+    ctrl,
+    measurement_grouping,
+    metric_tensor,
+)
```

Comment:
This will move all of these methods to be directly accessible from `qml`.

+1, I think that's a good idea.

Reply:
There maybe some slight naming confusion.
`qml.transfroms.adjoint` now could mean the `adjoint.py` module or the `adjoint` function.

Reply:
Oh good point. I think this import/module naming clash is common all over PennyLane.

Although, I think it is context dependent?

```python
from pennylane.transforms import adjoint # imports the function
from pennylane.transforms.adjoint import adjoint # imports the function from the module

qml.transforms.adjoint # the function
```